### PR TITLE
LPS-192298 Permissions to handling the "Channel Default" tab

### PR DESCRIPTION
**Pull request by jaydawu**
https://liferay.atlassian.net/browse/LPS-192298
Test plan: https://liferay.atlassian.net/browse/COMMERCE-12689

Resent from https://github.com/liferay-commerce/liferay-portal/pull/4340

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

File: modules/apps/account/account-test/src/testFunctional/tests/AccountRoles.testcase [hunk 1]
```diff
@@ -828,6 +828,201 @@ definition {
...
+			User.assignOrgRoleCP(
+				roleTitle = "Account Manager",
+				userScreenName = "useramsn");
+
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "useramea@liferay.com",
+				userLoginFullName = "useramfn useramln");
+		}
+
+		task ("Then can view account 1 listed with a link") {
```
Comment (baharturk) [inline modules/apps/account/account-test/src/testFunctional/tests/AccountRoles.testcase:1005]: After Account Manager role is given, the user can see both accounts and their defaults, please update these last two parts to assert it for both accounts, and use Control Panel to navigate to Accounts Admin.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSalesAgent.testcase [hunk 2]
```diff
@@ -401,6 +401,63 @@ definition {
...
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "sales.agent@liferay.com");
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/minium/account-management");
+		}
+
+		task ("Then can view only the unlinked account 1 listed") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSalesAgent.testcase:436]: Then can view only account 1 is listed and cannot click on it

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSalesAgent.testcase [hunk 3]
```diff
@@ -663,6 +720,303 @@ definition {
...
+			Account.openAccountsAdmin();
+
+			LexiconEntry.gotoEntry(rowEntry = "Commerce Account 1");
+
+			Account.assignUsers(
+				accountName = "Commerce Account 1",
+				assigneeName = "sales.agent@liferay.com");
+		}
+
+		task ("And the account addressed are added") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSalesAgent.testcase:756]: And an account address is added

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSalesAgent.testcase [hunk 4]
```diff
@@ -663,6 +720,303 @@ definition {
...
+			CommerceAccountChannelDefaults.setDefaultPriceList(
+				channelEntry = "Minium Portal",
+				priceListTitle = "Minium Base Price List");
+
+			AssertElementPresent(
+				key_entryDetails = "Minium Base Price List",
+				key_termType = "Price",
+				locator1 = "Account#DEFAULT_ACCOUNT_TERMS_AND_CONDITION_DROPDOWN");
+
+			Account.setDefaultCurrency(currencyName = "Euro");
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSalesAgent.testcase:858]: After you set as Euro, you make assertion for US Dollar below, it should be Euro I guess.

**Review by baharturk — CHANGES_REQUESTED** [review]
Please take a look at my comments. Thanks!

**Comment by jaydawu** [comment]
ci:test:bundle

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:um

**Comment by jaydawu** [comment]
@baharturk Since there are two tests written in the AccountRoles.testcase file, I run the ci:test:um to ensure the tests can be passed on CI as well.

**Review by baharturk — APPROVED** [review]

**Comment by jaydawu** [comment]
@baharturk I don't know why ([LocalFile.AccountRoles#CanUserWithAccountManagerRoleViewAndManageChannelDefaults](https://testray.liferay.com/home/-/testray/case_results/2452145652?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayCaseName%3DCanUserWithAccountManagerRoleViewAndManageChannelDefaults%26testrayBuildId%3D2452144582%26testrayRunId%3D2452144624%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc)) there is no Channel Defaults tab when CI ran, I rerun the test locally, it can be passed, maybe it's the cache issue on CI, I'd like to rerun the test suite, please help monitor when the result comes out, thank you!

**Comment by jaydawu** [comment]
ci:test:um

**Comment by jaydawu** [comment]
ci:test:um

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by smottal** [comment]
Resent here: https://github.com/liferay-commerce/liferay-portal/pull/4408

**Pull request closed without merging**